loopd/perms/perms.go

```diff
@@ -69,7 +69,7 @@ var RequiredPermissions = map[string][]bakery.Op{
 		Entity: "loop",
 		Action: "in",
 	}},
-	"/looprpc.SwapClient/GetLsatTokens": {{
+	"/looprpc.SwapClient/GetL402Tokens": {{
```

Comment:
Hmm, I didn't really think about this in my last review. But I think this might break some client applications (like Terminal Web, RTL or others that have Loop integrated) that rely on the old RPC name. To avoid headaches, I recommend to not rename this. Maybe we can add a new RPC and declare this one as deprecated, then remove it in a couple of releases.

Reply:
I added GetLsatTokens back as a wrapper. Updated release notes file.

Tested with old `loop` and new `loopd`:

```
$ loop listauth
[]
```

Loopd log:

```
2024-04-25 13:52:27.367 [WRN] LOOPD: Received deprecated call GetLsatTokens. Please update the client software. Calling GetL402Tokens now.
```